tests/integrations/android_sdk/AndroidSDK/test_detect_system_images.py

```diff
@@ -0,0 +1,63 @@
+import subprocess
+from unittest.mock import MagicMock, patch
+
+import pytest
+
+from briefcase.exceptions import BriefcaseCommandError
+from briefcase.integrations.android_sdk import AndroidSDK
+
+
+@pytest.fixture
+def android_sdk(android_sdk, tmp_path) -> AndroidSDK:
+    android_sdk.tools.subprocess.check_output = MagicMock(
+        return_value=(
```

Comment:
This test is also slightly misleading - it contains the "useful" part of the `sdkmanager` output, but not the pre-postamble. Since one of the purposes of this function is to strip the pre/postamble, we need to have a test of that capability.